### PR TITLE
Sjukratryggingar/status and children

### PR DESCRIPTION
**Pull request by tthornvik**
# Health insurance application - Status and children section

## What

Second section of the health insurance application. It contains:

- The current status of the applicant, pensioner, student and other
- The possibility for the applicant to upload a proof of studies 
- A way to tell if you are bringing children with you to island

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Screenshot 2020-12-10 at 09 28 02
Screenshot 2020-12-09 at 15 04 51
Screenshot 2020-12-09 at 15 05 01

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally 
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/health-insurance/src/fields/CurrentStatus/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    field.id,
+    undefined,
+  ) as ValidAnswers
+  const [statefulAnswer, setStatefulAnswer] = useState<ValidAnswers>(
+    currentAnswer,
+  )
+
+  return (
+    <Box marginY={3} key={field.id}>
+      <FieldDescription description="What is your current status?" />
```
Comment (dd-sdang) [inline libs/application/templates/health-insurance/src/fields/CurrentStatus/index.tsx:27]: Should perhaps use text from messages file? :)

File: libs/application/templates/health-insurance/src/forms/messages.ts [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+    description: 'Some description',
+  },
+  statusStudentInformation: {
+    id: 'pl.application:student.information',
+    defaultMessage:
+      'You moved away from Iceland for the purpose of studying abroad and are moving back to Iceland withing six months of the end of studies.',
+    description: 'Some description',
+  },
+  yesOptionLabel: {
+    id: 'pl.application:children.yes',
```
[the remark this answers is not in the document]
Comment (tthornvik) [reply libs/application/templates/health-insurance/src/forms/messages.ts:163]: Did not think of that, good comment.

File: libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts [hunk 3]
```diff
@@ -115,12 +118,55 @@ export const HealthInsuranceForm: Form = buildForm({
...
+                  tooltip: m.statusStudentInformation,
+                },
+                {
+                  label: m.statusOther,
+                  value: 'other',
+                  tooltip: m.statusOtherInformation,
+                },
+              ],
+            }),
+            buildFileUploadField({
```
Comment (arnifreyrs) [inline libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts:152]: note the file upload field is a bit buggy and might need some improvements before this goes live. Just a heads up

**Review by dd-sdang — APPROVED** [review]
Looks good

**Pull request merged**